src/confluent_kafka/src/confluent_kafka.c

```diff
@@ -1621,7 +1621,7 @@ static void oauth_cb (rd_kafka_t *rk, const char *oauthbearer_config,
         Py_ssize_t rd_extensions_size = 0;
         char err_msg[2048];
         rd_kafka_resp_err_t err_code;
-
```

Comment:
Put all the variable declarations together, then an empty line for the rest of the code.
i.e.;
```c
        Py_ssize_t rd_extensions_size = 0;
        char err_msg[2048];
        rd_kafka_resp_err_t err_code;
        int i;

        cs = CallState_get(h);
```